commands/board/listall.go

```diff
@@ -93,3 +95,37 @@ func ListAll(ctx context.Context, req *rpc.BoardListAllRequest) (*rpc.BoardListA
 
 	return list, nil
 }
+
+// TODO use a generic function instead of the two below once go >1.18 is adopted.
```

Comment:
Let me know if this isn't the case, I did not find a nice way to do it in 1.17 but I'm far from a go guru

Reply:
You did right it's not possible until we upgrade to go1.18+.